docs/references/react-router/overview.mdx

```diff
@@ -1,16 +1,16 @@
 ---
 title: Clerk React Router SDK
-description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your React Router application.
+description: Learn how to integrate Clerk into your React Router application using the Clerk React Router SDK.
 ---
 
 > [!WARNING]
 > The React Router SDK is currently in beta.
 
-Clerk makes it simple to add authentication to your React Router application. This reference lists the features and methods available in Clerk's React Router SDK.
+The Clerk React Router SDK is built on top of the [React SDK](/docs/references/react/overview) and is the recommended method for integrating Clerk into your React Router application.
 
 ## Client-side helpers
 
-The Clerk React Router SDK is built on top of the Clerk React SDK and provides access to all of Clerk React's hooks. These hooks give you access to the [`Clerk`](/docs/references/javascript/clerk/clerk) object and a set of useful helper methods for signing in and signing up. Learn more about these hooks in the [React SDK reference](/docs/references/react/overview).
```

Comment:
same note here about keeping this context.

Reply:
https://github.com/clerk/clerk-docs/pull/1815/commits/a8a1b5b1b89e127b992bb55417394db4181c6afc